Blazorade.Bootstrap.Components.Showroom/Host/Shared/MainLayout.razor

```diff
@@ -2,6 +2,6 @@
 
 <MainMenu />
 
-<Div class="container-fluid" MarginTop="Spacing.Four">
+<Div class="container-fluid" MarginTop="Spacing.Four" MarginLeft="Spacing.Five" Width="ComponentSize.p75">
```

Comment:
This will make the usable area pretty narrow on the smallest screens. An equal margin on both sides might be a better option, or what do you think?

```suggestion
<Div class="container-fluid" MarginTop="Spacing.Four" MarginX="Spacing.Three">
```

Reply:
I've experimented with it, and a PaddingX="Spacing.Three" will probably work better. Otherwise, we're pushing content off the screen to the right at full resolution.